static/docs/get-started/share-data.md

```diff
@@ -12,7 +12,9 @@ The same way as with Git remote, it ensures that your data files and your models
 are safely stored remotely and are shareable. It means that this data could be
 pulled by your team or you when you need it.
 
-Usually, you run it along with `git commit` and `git push` to save changed
+Usually, you run it along with `git commit` and
+[git push](https://git-scm.com/docs/git-push/1.5.5.2)
```

Comment:
no need in this link, tbh

Reply:
Shall I describe it here ? IMO there is no need to explain ``git push``

Reply:
I would change the paragraph, to emphasize that `dvc push` does not include or run `git push` and takes care only of data, that you still need to remember to run `git commit` and `git push`

Reply:
okay.